packages/oca/src/types/TypeEnums.ts

```diff
@@ -0,0 +1,20 @@
+export enum BaseType {
+  Binary = 'Binary',
+  Text = 'Text',
+  DateTime = 'DateTime',
+  Numeric = 'Numeric',
+  DateInt = 'DateInt',
+}
+
+export enum BaseOverlayType {
```

Comment:
I would just rename this to OverlayType we can also combine Branding into here

Reply:
I left it separated because they are not an "official" oca overlay. For bifold use, it makes easier to select which type of credential card was chosen. I'll make an "extra type" on legacy oca to keep it working, but combine overlay types seems to be a good idea.